src/media/css/app-list.styl

```diff
@@ -61,6 +62,14 @@ $desktop-tile-padding = 20px 50px;
     text-transform: uppercase;
 }
 
+.app-list.app-list-colored {
+    // When the app list has a colored background (e.g., Feed landing).
+    .mkt-tile h3,
+    .mkt-tile .author {
+        color: $white;
```

Comment:
Still find it funny that `#fff` is 2 chars shorter. Might as well be consistent though.